scripts/manpage_generation.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Directory to store the generated manpages
+MANPAGE_DIR="../man"
```

Comment:
If the scripts is being executed from the root folder (`convert2rhel`), then we don't need to specify the `..` as the `man` folder is already on the same level. The `..` is needed if we are executing this script inside the `scripts` folder, because `man` is on a level below this one. 


```suggestion
MANPAGE_DIR="man"
```

Reply:
I have gotten this completed!